src/main/java/it/pagopa/pn/client/b2b/pa/polling/dto/PnPollingResponseV23.java

```diff
@@ -1,21 +1,18 @@
 package it.pagopa.pn.client.b2b.pa.polling.dto;
 
-import it.pagopa.pn.client.b2b.pa.generated.openapi.clients.externalb2bpa.model.FullSentNotificationV23;
-import it.pagopa.pn.client.b2b.pa.generated.openapi.clients.externalb2bpa.model.NewNotificationRequestStatusResponseV23;
-import it.pagopa.pn.client.b2b.pa.generated.openapi.clients.externalb2bpa.model.TimelineElementV23;
-import it.pagopa.pn.client.b2b.pa.generated.openapi.clients.externalb2bpa.model.NotificationStatusHistoryElement;
-import it.pagopa.pn.client.b2b.webhook.generated.openapi.clients.externalb2bwebhook.model_v2_3.ProgressResponseElementV23;
+import it.pagopa.pn.client.b2b.pa.generated.openapi.clients.externalb2bpa.model.*;
 import lombok.Getter;
 import lombok.Setter;
+
 import java.util.List;
 
 
 @Getter
 @Setter
 public class PnPollingResponseV23 extends PnPollingResponse {
```

Comment:
Invece che sovrascrivere gli oggetti in questa classe bisogna creare un nuovo oggetto specifico per la notifica v24 (**e.g.** `PnPollingResponseV24`)